sematic/db/queries.py

```diff
@@ -221,6 +222,39 @@ def save_run(run: Run) -> Run:
     return run
 
 
+def get_job(job_name: str, job_namespace: str) -> Optional[Job]:
+    """Get a job by name and namespace (or None if it doesn't exist)"""
+    with db().get_session() as session:
+        return (
+            session.query(Job)
+            .filter(Job.name == job_name)
+            .filter(Job.namespace == job_namespace)
```

Comment:
I expect there will be many jobs piling up in the DB. Please add an index to the table on `name` and `namespace`. It can (should) be unique. This should be done in the original PR

Reply:
They are both part of the primary key, which is [automatically indexed](https://stackoverflow.com/a/42899045/2540669).

Reply:
Oops, missed that. Thank you!